src/loaders.ts

```diff
@@ -255,6 +255,23 @@ export class EtherscanABILoader implements ABILoader {
 
 export class EtherscanABILoaderError extends errors.LoaderError { };
 
+// Etherscan Contract Source API response
```

Comment:
```suggestion
/// Etherscan Contract Source API response
```

One more slash makes it into a typedoc-friendly docstring. :)